model/src/market/reason.rs

```diff
@@ -0,0 +1,23 @@
+/*
+ * Yagna Market API
+ *
+ * The version of the OpenAPI document: 1.6.1
+ *
+ * Generated by: https://openapi-generator.tech
+ */
+
+use serde::{Deserialize, Serialize};
+
+/// Generic Event reason information structure.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct Reason {
+    #[serde(rename = "message")]
+    pub message: String,
+}
```

Comment:
```suggestion
    pub message: String,
    #[serde(flatten)]
    custom: HashMap<String, Value>,
}
```
This would capture other custom properties. Everyone can deserialize Reason to his own structure with more typed fields

Reply:
done